### ISSUE TITLE
Providing a search option for exact phrases

### ISSUE THREAD

**Issue by xpurichan** [Automattic/simplenote-ios#270]
**Request:** 
To be able to search exact combinations of words or phrases in the Simplenote app.

**Summary:**
* Current if I search my notes for "WordPress website" then Simplenote will load all of my notes that include the words "WordPress" **and** "website"
   * It will omit any notes that do not include both of these words (as expected)
   * It will include results for any notes that include both of these words **even if the words are not adjacent to each other**. (Even if this outcome is not desired.)

The feature request is to allow the search bar to match exact phrases. For example, we would like to see an option to specifically search for the combination of words "WordPress website" and only load results where these two words appear together. This would allow us to remove any results that use the word "WordPress" and "website" on different parts of the page.

**Details:**
This request applies to all Simplenote versions across all devices.

This request was initially submitted in 1701233-zen

**Comment by Shtalin** [comment]
Work on iPhone. Close issue, please

**Comment by rachelmcr** [comment]
While testing on both iPhone XS and iPad Mini with Simplenote iOS 4.16.0.0 (TestFlight beta version) I came across this issue.

It was especially surprising when I searched for a phrase where one of the words only partially matched the note content. For example, searching the phrase `Simplenote ios` returned a note that had the words `Simplenote` and `scenarios` in it. This matches the current expected behavior but felt odd (and frustrating when I couldn't find a way to narrow the search to exact matches).
